manifests/config.pp

```diff
@@ -147,14 +142,7 @@
           content => template("${module_name}/address.yaml.erb"),
           notify  => Service['datastax-agent'],
       }
-  }
```

Comment:
Why are we removing lines 150 to 157? 

From what I can see, we are loosing the ability in the Puppet module of using `Log4JAuditWriter` ie. using Log4J as a log appender. 

I don't think these lines should be removed. Can you explain please?